src/dispatch/plugins/dispatch_slack/workflow.py

```diff
@@ -215,7 +215,7 @@ def handle_workflow_submission_event(
     named_params = []
     for i in form_data.keys():
         if i.startswith(RunWorkflowBlockIds.param_input):
-            key = i.split("-")[1]
+            key = i.split(RunWorkflowBlockIds.param_input + "-")[1]
```

Comment:
without this fix, the workflow message is sent in the key called "workflow"
After this fix, the workflow message is sent in the key called "message"